compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.9"
+services:
+   headnode:
+      image: mmmain/hadoop:headnode
+      container_name: headnode
+      volumes:
+         - hmnt1:/opt/mount1
+         - hmnt2:/opt/mount2
+         - hlogs:/usr/local/hadoop/current/logs
+#      ports:
```

Comment:
Should we uncomment this in order to allow users to easily reach WebUI?

Reply:
We can use a "localhost:8088" in the browser address field in this case (uncommented "ports" option)